src/test/java/core/basesyntax/service/operations/BalanceOperationTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private FruitTransaction transaction;
+
+    @Test
+    void apply_correctOperation_Ok() {
+        Storage.clear();
+        transaction = new FruitTransaction(
+                FruitTransaction.Operation.BALANCE,new Fruit("apple"),1);
+        OperationHandler operation = new BalanceOperation();
+        operation.apply(transaction);

```

Comment:
There is only one test case. Consider also testing for edge cases and invalid inputs to improve the test coverage.

Reply:
ok, done